dev/abbasidaniyal/generated.ipynb

```diff
@@ -0,0 +1,1807 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>std</th>\n",
+       "      <td>4.040587</td>\n",
+       "      <td>5.332695</td>\n",
+       "      <td>1.687016</td>\n",
+       "      <td>3.509535</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>min</th>\n",
+       "      <td>-4.400123</td>\n",
+       "      <td>-11.219695</td>\n",
+       "      <td>-4.276724</td>\n",
+       "      <td>-2.116425</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>25%</th>\n",
+       "      <td>-1.042873</td>\n",
+       "      <td>-6.659021</td>\n",
+       "      <td>0.341375</td>\n",
+       "      <td>2.004535</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>50%</th>\n",
+       "      <td>-0.020341</td>\n",
+       "      <td>3.021220</td>\n",
+       "      <td>1.769409</td>\n",
+       "      <td>6.388002</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>75%</th>\n",
+       "      <td>6.940277</td>\n",
+       "      <td>4.110598</td>\n",
+       "      <td>2.907363</td>\n",
+       "      <td>8.823314</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>max</th>\n",
+       "      <td>11.044036</td>\n",
+       "      <td>7.411504</td>\n",
+       "      <td>6.789665</td>\n",
+       "      <td>13.484701</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                  0             1             2             3\n",
+       "count  20000.000000  20000.000000  20000.000000  20000.000000\n",
+       "mean       2.057691      0.055582      1.629570      5.726769\n",
+       "std        4.040587      5.332695      1.687016      3.509535\n",
+       "min       -4.400123    -11.219695     -4.276724     -2.116425\n",
+       "25%       -1.042873     -6.659021      0.341375      2.004535\n",
+       "50%       -0.020341      3.021220      1.769409      6.388002\n",
+       "75%        6.940277      4.110598      2.907363      8.823314\n",
+       "max       11.044036      7.411504      6.789665     13.484701"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Normalizing the values of X and Splitting dataset into a training and a cross validation set\n",
+    "Doing a 60:40 split of the dataset for training and cross-validation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "temp =x.values #returns a numpy array\n",
+    "\n",
+    "min_max_scaler = preprocessing.MinMaxScaler()\n",
+    "x_scaled = min_max_scaler.fit_transform(temp)\n",
+    "\n",
+    "x_norm = pd.DataFrame(x_scaled)\n",
+    "x_train,x_test,y_train,y_test=train_test_split(x_norm,y,test_size=0.6)\n"
```

Comment:
You are using 40% training, 60% testing. This is valid, but I'm not sure if that's what you intended given your comment above?